### PR TITLE
Add consistent code quoting

### PR DESCRIPTION
**Pull request by wwwutz**
- Move shell commands in a triple-backtick section.
- fix autonumbering

### CODE REVIEW

**Comment by wwwutz** [comment]
Before:
fix-code-before

After:
fix-code-after

File: general/administration/building.md [hunk 1]
```diff
@@ -12,29 +12,37 @@ Jellyfin supports several methods of building for different platforms and instru
 All package builds begin with these two steps:
 
 1. Clone the repository:
-    `git clone https://github.com/jellyfin/jellyfin.git`<br/>
-    `cd jellyfin`
+    ```sh
+    git clone https://github.com/jellyfin/jellyfin.git
+    cd jellyfin
+    ```
```
Comment (dkanada) [inline general/administration/building.md]: What will the indentation look like in DocFX? Will code blocks even respect indentation?

Comment (wwwutz) [reply general/administration/building.md]: yes, they will. I fed i to `docfx` through a docker image and it did what I expected.

Comment (wwwutz) [reply general/administration/building.md]: docfxindent

Comment (dkanada) [reply general/administration/building.md]: Since we're making changes to these code blocks, can we add newlines before and after as well?

**Comment by wwwutz** [comment]
Yes we can add more newlines, as long as the code quote is indented:

No empty newline behind number:

num-no-newline

With empty newlines behind number:

num-with-newline png

BUT we have to have an empty newline  between numbered paragraphs, otherwise this will happen:

num-no-newline-at-all

This is why I added them in `general/clients/installing-mopidy.md`. Was quite a struggle to find out why code quoting didn't work there.

**Comment by dkanada** [comment]
Alright cool, let's add newlines when possible then.

**Comment by wwwutz** [comment]
anything I should change/edit/fix ?

**Comment by Artiume** [comment]
Looks good, thanks for all the effort. Not for this PR, but what are your thoughts of making the guide using pipenv instead of pip, that way mopidy doesn't have to monkey with the system pip? I haven't used mopidy so I don't know if it'd have any effect on it.

**Comment by wwwutz** [comment]
> your thoughts of making the guide using pipenv instead of pip,

none. not even one neuron gets triggered.

> I haven't used mopidy ...

me neither. I might take a look at it, but I doubt  I'll dig into that deeper.

**Comment by Artiume** [comment]
Haha, not a fan?

**Pull request merged**